setup.py

```diff
@@ -293,6 +293,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'hmsclient>=0.1.0',
     'pyhive[hive]>=0.6.0',
 ]
+impala = ["impyla>=0.16.3"]
```

Comment:
Any particular reasoning for this dependency range?

Reply:
@uranusjr I assume it has to do with peer-dependencies mentioned [here](https://github.com/apache/airflow/pull/8918)